ios/RNFIRMesssaging.m

```diff
@@ -99,6 +99,14 @@ - (void)setBridge:(RCTBridge *)bridge
   [[NSNotificationCenter defaultCenter]
    addObserver:self selector:@selector(onTokenRefresh)
    name:kFIRInstanceIDTokenRefreshNotification object:nil];
+
+  [[NSNotificationCenter defaultCenter]
```

Comment:
Hi @evollu,

They are not `mandatory`, but would be `nice to have`. They are just event listeners to know if the message was successfully delivered on the app server or not. They can also be useful when debugging with XCode console. You can find further documentation at https://developers.google.com/cloud-messaging/ios/ios-migrate-fcm and https://firebase.google.com/docs/cloud-messaging/ios/upstream.

Cheers